convert2rhel/toolopts.py

```diff
@@ -30,7 +31,7 @@ def __init__(self):
         self.enablerepo = []
         self.disablerepo = []
         self.pool = None
-        self.variant = None
+        self.variant = "Server"
```

Comment:
We should remove it completely. For all usages of variant we should just replace it with the string `server`
```suggestion
```

Reply:
Nit: This string "Server" is a constant, probably it has the sense to move it to the top of the module. (i.e.
```python
DEFAULT_VARIANT = "Server"
```

Reply:
We use `toolopts.variant` in two spots I believe, I don't believe it's worth keeping the variable as we'll always use server